pkg/oc/cli/admin/network/join_projects.go

```diff
@@ -37,7 +37,7 @@ type JoinOptions struct {
 	joinProjectName string
 }
 
-func NewCmdJoinProjectsNetwork(commandName, fullName string, f kcmdutil.Factory, out io.Writer) *cobra.Command {
+func NewCmdJoinProjectsNetwork(commandName, fullName string, f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Command {
 	opts := &ProjectOptions{}
 	joinOp := &JoinOptions{Options: opts}
```

Comment:
Same thing as above comment - wrap the ProjectOptions Complete and Validate methods, and set the IOStreams here instead of passing them as an argument of the Complete method